test/command/suite/column_copy/float32.test

```diff
@@ -0,0 +1,19 @@
+table_create Tags TABLE_HASH_KEY ShortText
```

Comment:
Could you rename this file to `with_float32.test`?

Reply:
I see.
I fixed file name at https://github.com/groonga/groonga/pull/1379/commits/7b1ab445b36ec86ae2446fc2309e6f30d6137dbc.

Reply:
Sorry. `weight_float32.test`...

Reply:
Oh.
I fixed the file name again at https://github.com/groonga/groonga/pull/1379/commits/4da11d7f21459503c5e76076fd31d32d56d86ea5.